apps/data-types.md

```diff
@@ -251,6 +259,31 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | lineItemPrice | object of [price](page:apps-data-types#price) | The price of the line item. |
 | lineItemProductGuid | string | The identifier of the related product line item. A deposit is always bound to a product.|
 
+## facet
+
+| Attribute | Type | Description |
+| - | - |  - |
+| attributeName | string | The name of the filter. Can be *ListPrice*, *Manufacturer*, *CategoryID*, or any other attribute created by the merchant.  |
+| attributeType | string | xxx |
+| values | array of [filterValues](page:apps-data-types#filterValues) | The values to refine the search. |
+
+## filter
+
+| Attribute | Type | Description |
+| - | - |  - |
+| attributeValue  | string | The value of the facet. |
+| attributeRange  | object of [attributeRange](page:apps-data-types#attributerange) | The minimum and maximum range of the filter applied. |
+| fieldName  | string | The alias of the filter.  |
+
+## filterValues
+
+| Attribute | Type | Description |
+| - | - |  - |
+| matches | string | xxx |
+| name | string | xxx |
+| filter | object of [filter](page:apps-data-types#filter) | xxx |
+| selectedValue | string | xxx |
```

Comment:
This is the selected attribute value used as filter. (This value was given used in the input as filter value)